src/main/java/seedu/address/ui/UiManager.java

```diff
@@ -44,6 +47,15 @@ public void start(Stage primaryStage) {
             mainWindow.show(); //This should be called before creating other UI parts
             mainWindow.fillInnerParts();
 
+            //show notifications
+            Notifications notificationBuilder = Notifications.create()
+                    .title("Test Notification")
+                    .text("This is a test Notification")
+                    .graphic(null)
+                    .hideAfter(Duration.seconds(5))
+                    .position(Pos.TOP_RIGHT);
+            notificationBuilder.showConfirm();
+
```

Comment:
Can be abstracted out to provide a standardised notification api for internal use.
e.g. standardaise ui, position